bundle/regal/lsp/completion/providers/booleans/booleans.rego

```diff
@@ -0,0 +1,39 @@
+package regal.lsp.completion.providers.booleans
+
+import rego.v1
+
+import data.regal.lsp.completion.kind
+import data.regal.lsp.completion.location
+
+parsed_current_file := data.workspace.parsed[input.regal.file.uri]
+
+items contains item if {
+	position := location.to_position(input.regal.context.location)
+
+	line := input.regal.file.lines[position.line]
+	line != ""
+
+	words := regex.split(`\s+`, line)
+
+	words_on_line := count(words)
+
+	previous_word := words[words_on_line - 2]
+
+	previous_word in {"==", ":="}
+
+	word := location.word_at(line, input.regal.context.location.col)
+
+	some b in ["true", "false"]
+
+	startswith(b, word.text)
```

Comment:
```
some b in ["true", "false"]

startswith(b, word.text)
```

can be replaced by `strings.any_prefix_match`, I think

Reply:
I guess I need to know if we're going to be using true or false in the creation of the item though?